app/components/press-release/form.hbs

```diff
@@ -78,3 +97,11 @@
     </div>
   </div>
 </div>
+{{#if this.showSourceModal}}
+  <PressRelease::SourcesModal
+    @sources={{@pressRelease.sources}}
+    @onCancel={{this.closeSourceModal}}
+    @onChange={{this.setSources}}
+    @loading={{this.setSources.isRunning}}
```

Comment:
`this.setSources` is een `@action` een geen `@task` dus die heeft geen `isRunning` property. Aangezien er niets async gebeurt in de method, lijkt die `loading` state mij hier sowieso overbodig.